dockers/docker-database/base_image_files/monit_database

```diff
@@ -4,4 +4,4 @@
 ##  redis_server
 ###############################################################################
 check program database|redis_server with path "/usr/bin/process_checker database /usr/bin/redis-server"
-    if status != 0 for 5 times within 5 cycles then alert
+    if status != 0 for 5 times within 5 cycles then alert repeat every 5 cycle
```

Comment:
I believe the proper syntax is "repeat every X **cycles**", plural. This applies to all the files in this PR.

Also, I believe we should repeat the alert every cycle until it is resolved.

Reply:
@jleveque Both "cycles" and "cycle are supported.

Updated to use repeat every 1 cycles

Reply:
>@jleveque Both "cycles" and "cycle are supported.

Good to know!